tests/safety_test.py

```diff
@@ -1,52 +1,66 @@
 # pylint: disable=invalid-name,line-too-long
-from __future__ import absolute_import
-from __future__ import unicode_literals
-
 import pytest
 from safety.constants import EXIT_CODE_VULNERABILITIES_FOUND
 
 from pre_commit_hooks.safety_check import main as safety
 
 
 def test_dev_requirements():
-    assert safety(['dev-requirements.txt']) == 0
+    assert safety(["dev-requirements.txt"]) == 0
+
+
+def test_non_ok_dependency(tmp_path):
+    requirements_file = tmp_path / "requirements.txt"
+    with open(requirements_file, "w") as file:
```

Comment:
```suggestion
    with open(requirements_file, "w", encoding="utf8") as file:
```
Please always specify the encoding when calling `.open()`

Reply:
Sure